### PR TITLE
add more error logging

### PR DESCRIPTION
**Pull request by mdroidian**

### CODE REVIEW

File: api/oauth/post.ts [hunk 1]
```diff
@@ -40,11 +47,19 @@ const logic = async (
...
+      console.error(e);
+      console.error("Arguments:");
+      const maskedArgs = {
+        ...args,
+        code: maskString(args.code),
+      };
+      console.error(maskedArgs);
+      return Promise.reject(
         new Error(`Failed to get access token: ${e.response.data}`)
-      )
```
Comment (dvargasfuertes) [inline api/oauth/post.ts]: let's replace this `Promise.reject` with a ` throw new ServerError('...', 401)`

File: api/oauth/post.ts [hunk 2]
```diff
@@ -40,11 +47,19 @@ const logic = async (
...
-    );
+      );
+    });
   const { access_token } = data;
   const privateKey = process.env.APP_PRIVATE_KEY;
   const workspace = privateKey
```
Comment (dvargasfuertes) [inline api/oauth/post.ts:66]: let's do the same thing on the Octokit api call below

**Review by dvargasfuertes — APPROVED** [review]
I have ideas on how we could replace `axios` with our own `fetch` implementation that adds records to `Amazon Redshift` so that we could have granular logging going forward

File: api/oauth/post.ts (hunk 2 above)
Comment (mdroidian) [reply api/oauth/post.ts:66]: nice call 🙌

image

**Pull request merged**